<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <widget class="QPushButton" name="closedButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/shareImages/close.png);
}

QPushButton:pressed{
border-image: url(:/shareImages/close_press.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="loginInfoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>135</y>
     <width>303</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>296</width>
     <height>301</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font-size:12px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>18</number>
    </property>
    <property name="leftMargin">
     <number>18</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>18</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>86</x>
          <y>34</y>
          <width>95</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/shareImages/logo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>23</y>
          <width>28</width>
          <height>33</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>33</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#label {	
	border-image: url(:/loginImage/lock.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="numberLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:14px;
color: rgb(51, 51, 51);</string>
          </property>
          <property name="text">
           <string>账号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userNameEdit">
          <property name="minimumSize">
           <size>
            <width>205</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/shareImages/editRect.png);
padding-left:10px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="storeNumberLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:14px;
color: rgb(51, 51, 51);</string>
            </property>
            <property name="text">
             <string>密码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordEdit">
            <property name="minimumSize">
             <size>
              <width>205</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/shareImages/editRect.png);
color: rgb(51, 51, 51);
padding-left:10px;
font:8px;</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="hintLabel">
            <property name="minimumSize">
             <size>
              <width>205</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#fa7a7a;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:#ffffff;
font-size:16px;
border-image: url(:/loginImage/commonButton.png);
}

QPushButton:pressed{
border-image: url(:/loginImage/commonButtonPressed.png);
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>2</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="WndDigitalKeyboard" name="keyboardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>135</y>
     <width>302</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>302</width>
     <height>301</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="splitLine">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>2</width>
     <height>600</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header>WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
